dev_guide/templates.adoc

```diff
@@ -12,35 +12,40 @@ toc::[]
 == Overview
 A template describes a set of
 link:../architecture/core_objects/overview.html[resources] that can be
-customized and processed to produce a configuration. Each template is a
-parameterized list that is used by OpenShift to create a list of resources,
-including services, pods, routes, buildconfigs, etc. A template also defines a
-set of labels to apply to every resource created by the template.
+parameterized and processed to produce a list of resources that are meant to be
+created by OpenShift.
```

Comment:
s/that are meant to be created by OpenShift/for creation by OpenShift/

Maybe?